src/mavedb/routers/score_sets.py

```diff
@@ -153,7 +165,17 @@ def get_score_set_scores_csv(
 
     columns = ["accession", "hgvs_nt", "hgvs_splice", "hgvs_pro"] + score_set.dataset_columns["score_columns"]
     type_column = "score_data"
-    rows_data = get_csv_rows_data(score_set.variants, columns=columns, dtype=type_column)
+    variants = score_set.variants
+
```

Comment:
something like 
variants = variants[start if start else 0:]
if limit:
  variants = variants[:limit]

is a little cleaner imo

Reply:
Thanks for your feedback. I tried it. It can't handle correctly if both `start` and `limit` are present. The reason is `variants` is cut three times. The condition still needs clarify if users only type either `start` or `limit`.

> something like if start: variants = variants[start:] if limit: variants = variants[:limit]
> 
> is a little cleaner imo

Reply:
@ashsny In my email I see your comments “comment says variants, should probably be counts”, but I don’t see it in GitHub. I don't know why. 
I leave comments of both `limit` of scores and counts are number of variants. The reason is both of them belong to variants. Some variants have both and some have only scores but no count data. The `limit` number returns the number of variants, but not the number of counts or scores. I don't think we need part of counts or scores. That will be huge amount of data again. It's easy to understand their relationship in Table `variants` Column `data`.

Reply:
I'm not sure what you mean by "`variants` is cut three times." Is there a specific case you think won't be handled correctly by the code I posted? Maybe I am mis-understanding the API here. 

see examples:
https://colab.research.google.com/drive/17tEMiKutF8Wrh2AnA9rdWBQ7mR7lXbGz?usp=sharing

Reply:
Sorry, my fault. I thought these codes are still there. 
```
if start and limit:
        end = start + limit
        variants = variants[start:end]
```
Yes, yours is better. I'll change these parts and push again.